metricbeat/docs/modules/elasticsearch.asciidoc

```diff
@@ -5,7 +5,7 @@ This file is generated! See scripts/mage/docs_collector.go
 [[metricbeat-module-elasticsearch]]
 == Elasticsearch module
 
-The `elasticsearch` module collects metrics about {es}.
+The `elasticsearch` module collects metrics about {es}. In order to get all metricsets, you must install Metricbeat on each Elasticsearch node in the monitored cluster.
```

Comment:
This is not quite true. Metricbeat could be installed anywhere and this module's `hosts` settings could be configured to point to an Elasticsearch cluster's nodes.

Also, recently we introduced a `scope: cluster` setting that let's users point this module to _any_ node in the Elasticsearch cluster: https://github.com/elastic/beats/pull/18547.